build_tools/packager/gem_packager.rb

```diff
@@ -10,11 +10,20 @@ def initialize
 
     def build
       Dir.chdir @repo_root do
-        gem = GemPublisher::Builder.new.build('govuk_template.gemspec')
+        stdout_str = run('gem build govuk_template.gemspec')
+        gem = stdout_str[/File:\s+(.+)/, 1]
 
         @repo_root.join('pkg').mkpath
         FileUtils.mv(gem, "pkg")
       end
     end
+
+    private
```

Comment:
GDS style is [to outdent `private`](https://github.com/alphagov/service-manual-publisher/blob/master/app/forms/guide_form.rb#L8)